### PR TITLE
feat(thorium): add remaining os support

### LINKED ISSUE #2: Add Thorium Browser
as the title says

it's a based fork of chromium becoming popular on gnu linux

### PR DESCRIPTION
**Pull request by BrycensRanch**
# I'm back with the milk 🥛

- Adds detecting Thorium on ABSOLUTELY PROPRIETARY MICROSOFT WINDOWS
- Adds detecting Thorium on Windows fearing operating System big mac os

Closes #2 

image

### CODE REVIEW

**Comment by BrycensRanch** [comment]
ok see you next year when this gets merged 
image

File: macos.js [hunk 1]
```diff
@@ -29,15 +29,18 @@ function getEdge() {
...
 }
 
 function getFirefox() {
 	return testPath("/Applications/Firefox.app/Contents/MacOS/firefox");
 }
 
 
-module.exports = { getChrome, getEdge, getChromium, getFirefox };
\ No newline at end of file
+module.exports = { getChrome, getEdge, getChromium, getFirefox };
```
Comment (albertopasqualetto) [inline macos.js]:
```suggestion
module.exports = { getChrome, getEdge, getThorium, getChromium, getFirefox };
```

File: windows.js [hunk 2]
```diff
@@ -46,4 +50,4 @@ function getFirefox() {
 }
 
 
-module.exports = {getChrome, getEdge, getChromium, getFirefox};
\ No newline at end of file
+module.exports = {getChrome, getEdge, getChromium, getFirefox};
```
Comment (albertopasqualetto) [inline windows.js]:
```suggestion
module.exports = {getChrome, getEdge, getThorium, getChromium, getFirefox};
```

**Comment by albertopasqualetto** [comment]
So welcome to 2024!
(And thanks for your PR)
Have a nice day

**Comment by BrycensRanch** [comment]
You too! thanks for fixing my mistake, forgot to export the new function.

**Pull request merged**